vagrant/manifests/default.pp

```diff
@@ -0,0 +1,26 @@
+node 'jenkins-master' {
+  include jenkins_master
+
+  class { 'web::base':
+    letsencrypt => false,
+  }
+
+  class { 'web::jenkins':
+    hostname => $facts['fqdn'],
+  }
+
+  package { 'python-pip':
```

Comment:
`ensure_packages(['python-pip'])`

Reply:
Did this in the JJB module since it's not only needed in the vagrant case.